src/lib/menu/menu-item.ts

```diff
@@ -105,6 +112,7 @@ export class MatMenuItem extends _MatMenuItemMixinBase
   /** Gets the label to be used when determining whether the option should be focused. */
   getLabel(): string {
     const element: HTMLElement = this._elementRef.nativeElement;
+    const textNodeType = this._document ? this._document.TEXT_NODE : 3;
```

Comment:
Make constants for node types until `_document` is always available?

Reply:
I'm inclined not to, because we'll end up sprinkling them everywhere and it'll make it more likely to forget about removing them.